lib/wanda/results.ex

```diff
@@ -3,33 +3,37 @@ defmodule Wanda.Results do
   This module exposes functionalities to interact with the historycal log of ExecutionResults.
   """
 
+  alias Wanda.Repo
+
   alias Wanda.Execution.Result
   alias Wanda.Results.ExecutionResult
 
   import Ecto.Query
 
   @doc """
-  Saves a result to the database.
+  Create a new result.
   """
-  @spec save_result(Result.t()) :: ExecutionResult.t()
-  def save_result(
+  @spec create_execution_result(Result.t()) :: ExecutionResult.t()
+  def create_execution_result(
         %Result{
           execution_id: execution_id,
           group_id: group_id
         } = result
       ) do
-    Wanda.Repo.insert!(%ExecutionResult{
+    Repo.insert!(%ExecutionResult{
```

Comment:
Shouldn't have Credo warned us about this?

Reply:
Credo warns only if it is nested 2 or 3 levels. 
Sometimes it is useful to alias with one level of nesting, for example when you have overlapping module names from different "contexts". This is not the case, tho.